aiida_quantumespresso/workflows/pdos.py

```diff
@@ -219,14 +222,15 @@ def define(cls, spec):
             help='Terminate workchain steps before submitting calculations (test purposes only).'
         )
         spec.input(
-            'align_to_fermi',
-            valid_type=orm.Bool,
+            'fermi_energy_range',
+            valid_type=orm.List,
+            required=False,
             serializer=to_aiida_type,
-            default=lambda: orm.Bool(False),
+            default=lambda: orm.List(list=[]),
```

Comment:
Don't think this needs a default if it is going to be empty anyway.

Reply:
I agree, I will remove this